packages/main/src/Carousel.hbs

```diff
@@ -24,13 +24,15 @@
         {{#if arrows.content}}
             <div class="ui5-carousel-navigation-arrows">
                 <ui5-button
+                    arrow-back
```

Comment:
Do we need these?

Reply:
it is used in the test to retrieve the backward arrow in more explicit way, but I guess ui5-carousel-navigation-arrows[0] will work as well, I can change it if you want